packages/did-eth-registry/contracts/DIDRegistry.sol

```diff
@@ -39,19 +32,13 @@ contract EthereumDIDRegistry {
         return identity;
     }
 
-    function checkSignature(
-        address identity,
-        uint8 sigV,
-        bytes32 sigR,
-        bytes32 sigS,
-        bytes32 digest
-    ) internal returns (address) {
-        address signer = ecrecover(digest, sigV, sigR, sigS);
-        require(signer == identityOwner(identity), "bad_signature");
-        nonce[signer]++;
-        return signer;
-    }
-
+    /**
+     * Return the current delegate of an identity for a specific delegate type.
+     * @param identity The identity to check.
+     * @param delegateType The type of the delegate.
+     * @param delegate The address of the delegate.
+     * @return bool The address of the current delegate.
+     */
```

Comment:
```suggestion
    /**
     * Determines if a specific address is currently listed as a delegate for a given identity and type of delegation.
     * @param identity - The identity to check.
     * @param delegateType - The type of the delegation being checked.
     * @param delegate - The address of the delegate.
     * @return bool - true if the delegation is currently valid, false otherwise.
     */
```